crates/rust-analyzer/tests/slow-tests/testdir.rs

```diff
@@ -10,7 +10,7 @@ pub(crate) struct TestDir {
 }
 
 impl TestDir {
-    pub(crate) fn new() -> TestDir {
+    pub(crate) fn new(symlink: bool) -> TestDir {
```

Comment:
nit, make this constructor private and expose two new ones without the bool param `new` and `new_symlinked` or similar that wrap the private one.